src/include/wiredtiger.in

```diff
@@ -752,6 +752,11 @@ struct __wt_cursor {
  *
  * <b>Thread safety:</b> A WT_SESSION handle is not usually shared between
  * threads, see @ref threads for more information.
+ *
+ * The operations requiring exclusive access to the btree, such as drop,
+ * verify, upgrade, salvage, rename, and alter, return EBUSY if the btree is
+ * dirty. Keep doing checkpoints to clean the btree before retrying these
+ * operations.
```

Comment:
I really think this has to be in the individual command text. Otherwise, it's just going to get lost. Could we incorporate common wording in a new ebusy return wording, maybe `ebusy_errors_checkpoint`?

BTW, note there are 2 Doxyfiles in src/docs, both would need changes if you go in that direction.